pybullet build time: Jan 18 2022 16:07:40
Traceback (most recent call last):
  File "/Users/Alexandre_1/Desktop/CS206/cs206/simulate.py", line 27, in <module>
    simulation.Run()
  File "/Users/Alexandre_1/Desktop/CS206/cs206/simulation.py", line 32, in Run
    self.robot.Sense(i)
  File "/Users/Alexandre_1/Desktop/CS206/cs206/robot.py", line 36, in Sense
    sensor.Get_Value(i)
  File "/Users/Alexandre_1/Desktop/CS206/cs206/sensor.py", line 13, in Get_Value
    self.values[i] = pyrosim.Get_Touch_Sensor_Value_For_Link(self.linkName)
  File "/Users/Alexandre_1/Desktop/CS206/cs206/pyrosim/pyrosim.py", line 53, in Get_Touch_Sensor_Value_For_Link
    for pt in pts:
TypeError: 'NoneType' object is not iterable
Exception ignored in: <function SIMULATION.__del__ at 0x104fcba30>
Traceback (most recent call last):
  File "/Users/Alexandre_1/Desktop/CS206/cs206/simulation.py", line 46, in __del__
pybullet.error: Not connected to physics server.
